### PR TITLE
update article embed URL for MediaPlayer stories

### LINKED ISSUE #2246: MediaPlayer articles is broken on Storybook
**Describe the bug**
https://bbc.github.io/psammead/?path=/story/components-media-player--amp
https://bbc.github.io/psammead/?path=/story/components-media-player--default
Are both failing to render as the embed URL has changed.

**Expected behaviour**
The above links should have real AV playback. 

**Screenshots**
image

**Testing notes**

- ~[ ] This bug fix is expected to need manual testing.~

**Additional context**

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #2246

**Overall change:** Updates the embed URL in the MediaPlayer stories

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~[] Automated jest tests added (for new features) or updated (for existing features)~
- ~[ ] This PR requires manual testing~

http://localhost:8180/?path=/story/components-media-player--default
http://localhost:8180/?path=/story/components-media-player--amp

https://bbc.github.io/psammead/?path=/story/components-media-player--amp
https://bbc.github.io/psammead/?path=/story/components-media-player--default

### CODE REVIEW

**Comment by pjlee11** [comment]
image

File: packages/components/psammead-media-player/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.1.1-alpha.2 | [PR#xxx](https://github.com/bbc/psammead/pull/xxx) Update article embed URLs in MediaPLayer stories |
```
Comment (pjlee11) [inline packages/components/psammead-media-player/CHANGELOG.md]:
```suggestion
| 1.1.1-alpha.2 | [PR#2247](https://github.com/bbc/psammead/pull/2247) Update article embed URLs in MediaPLayer stories |
```

**Review by dr3 — APPROVED** [review]

**Comment by pjlee11** [comment]
image

**Review by OlgaLyubin — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]
lgtm

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**